commerce_coordinator/apps/stripe/pipeline.py

```diff
@@ -79,16 +74,22 @@ class GetStripeDraftPayment(PipelineStep):
     Retrieve a PaymentIntent from Stripe.
     """
 
-    def run_filter(self, payment_data, **kwargs):  # pylint: disable=arguments-differ
+    def run_filter(self, **kwargs):
         """
         Executes a filter with the signature specified.
 
         Args:
-            payment_data (dict): The payment object.
+            kwargs['payment_data'] (dict): The payment object.
+            kwargs['payment_intent_data'] (dict): Optional. If truthy, skip this pipeline step.
         """
-        # Skip talking to Stripe if payment_intent_data is already populated.
+        # Payment intent already retrieved:
         if kwargs.get('payment_intent_data'):
-            return {}  # Keep pipeline unchanged.
+            return {}  # Skip pipeline step.
+
+        # No existing payment:
+        payment_data = kwargs.get('payment_data')
+        if not payment_data:
+            return {}  # Skip pipeline step.
```

Comment:
- [ ] Add constant for `{}` (to skip pipeline step) and `None` (to halt pipeline)